redaxo/src/addons/media_manager/lib/effects/effect_workspace.php

```diff
@@ -28,10 +28,10 @@ public function __construct()
 
 (function($) {
     $(function() {
```

Comment:
vllt geschmacksache, aber ich finde diese notation hier verwirrend

```
(function($) {
     $(function() {
```

sie hat eine daseinsberechtigung, aber könnte weniger beschlagene javascript entwickler verwirren... ggf. sollte man hier was lesbareres wählen

Reply:
Das erste Konstrukt wird soweit ich weiß nicht mehr benötigt, da wird jQuery in R5 nicht mehr im noConflict-Modus laufen lassen.